crates/robonode-server/src/logic/mod.rs

```diff
@@ -20,6 +20,9 @@ pub mod traits;
 
 pub use traits::*;
 
+/// A type alias representing a scan result blob.
+pub(crate) type ScanResultBlob = Option<String>;
```

Comment:
```suggestion
pub(crate) type ScanResultBlob = String;
```

Applying option like this is very confusing when it comes to reading the code, better apply it at the usage location: `Option<ScanResultBlob>`

Reply:
https://github.com/humanode-network/humanode/pull/1037/commits/33f49d52f6000f24f1c3ffb850fa4a23a8e86cd7